packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -92,6 +93,21 @@ Future<void> buildWindows(WindowsProject windowsProject, BuildInfo buildInfo, {
   } finally {
     status.stop();
   }
+
+  final String? binaryName = getCmakeExecutableName(windowsProject);
+  final File appFile = buildDirectory
+    .childDirectory('runner')
+    .childDirectory(sentenceCase(buildModeName))
+    .childFile('$binaryName.exe');
+  final String appSize = (buildInfo.mode == BuildMode.debug)
+      ? '' // Don't display the size when building a debug variant.
+      : ' (${getSizeAsMB(appFile.lengthSync())})';
+  globals.logger.printStatus(
+    '${globals.logger.terminal.successMark}  '
+    'Built ${globals.fs.path.relative(appFile.path)}$appSize.',
+    color: TerminalColor.green,
+  );
```

Comment:
This matches Android's behavior: https://github.com/flutter/flutter/blob/61a2f9de51007999c8a3f50e98061d7138a694eb/packages/flutter_tools/lib/src/android/gradle.dart#L501-L507